app/models/misc_item.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+class MiscItem < ApplicationRecord
+  belongs_to :game
+  belongs_to :canonical_misc_item,
+             optional: true,
+             inverse_of: :misc_items,
+             class_name: 'Canonical::MiscItem'
+
+  validates :name, presence: true
+  validates :unit_weight, numericality: { greater_than_or_equal_to: 0, allow_nil: true }
+  validate :validate_association
+
+  before_validation :set_canonical_misc_item
+
+  DUPLICATE_MESSAGE = 'is a duplicate of a unique in-game item'
+  DOES_NOT_MATCH = "doesn't match any item that exists in Skyrim"
+
+  def canonical_models
+    return [] if name.blank?
+    return [canonical_misc_item] if canonical_misc_item.present?
+
+    canonicals = Canonical::MiscItem.where('name ILIKE ?', name)
+
+    attributes_to_match.any? ? canonicals.where(**attributes_to_match) : canonicals
```

Comment:
This is kind of overkill when we could do:
```ruby
unit_weight.present? ? canonicals.where(unit_weight:) : canonicals
```

I chose not to do this, though, in order to make the API consistent with other canonical models for easier refactoring when we get to [this card](https://trello.com/c/EIdSrLs6/317-revisit-conditional-assignment-of-canonical-models).